app/controllers/users.py

```diff
@@ -1464,5 +1464,137 @@ def post(self):
             status='success',
             message=f'Successfully sent {emails_sent} reminder emails',
             total_users_processed=len(inactive_users),
-            errors=errors if errors else None
-        ), 201
+            errors=errors if errors else None)
```

Comment:
Please add back the status code
```suggestion
            errors=errors if errors else None), 201
```